### DISCUSSION TITLE
大家的百度视频播放正常吗

### DISCUSSION THREAD

**Discussion by xuze1993** [AlistGo/alist#6374] [General]
似乎视频一大播放器就会显示重新连接，大概50M以下的视频是正常的。
挂载用的本地代理，机器带宽和性能应该没什么问题
![316893049-db765385-e5e1-4138-b671-4ce26b52cfaf](https://github.com/alist-org/alist/assets/35798540/b532fc97-ac3f-43f2-9ab1-86be79779975)

**Comment by IDiDidiDiDidi** [comment]
一样，大点的也无法播放
